src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -484,12 +484,14 @@ FreezeBase::FreezeBase(JavaThread* thread, ContinuationWrapper& cont, intptr_t*
 
   assert(_cont.chunk_invariant(), "");
   assert(!Interpreter::contains(_cont.entryPC()), "");
-  static const int doYield_stub_frame_size = frame::metadata_words;
+  static const int doYield_stub_frame_size = NOT_PPC64(frame::metadata_words)
+                                             PPC64_ONLY(frame::abi_reg_args_size >> LogBytesPerWord);
```

Comment:
Unfortunate that we still need to distinguish, here. But, ok. I don't have a better idea atm.

Reply:
A pd constant could be introduced. Not sure if it's worth it.